api/app/routes.py

```diff
@@ -1,13 +1,19 @@
 from . import app
 from . import models
+from config import TOKEN, MEMBERSHIP_API
+import json
+import requests
 
-@app.route('/api/user', methods=['GET'])
-def getAllVoters():
-    return json.dump(User.query.all())
+@app.route('/api/user/login', methods=['POST'])
+def getVoterlogin():
 
-@app.route('/api/user/<int:uuid>', methods=['GET'])
+    r = request.get(MEMBERSHIP_API + "", headers=TOKEN)
```

Comment:
That TOKEN change reverberates a bit, but here's how you fix it:
```suggestion
    r = requests.get(MEMBERSHIP_API + "", headers = {"Authorization": f"Bearer {TOKEN}"})
```